lib/python/test/testsuite/_checkData.py

```diff
@@ -0,0 +1,222 @@
+Authors: Mika Soren Voss
+License: GPLv3+
+
+This module checks whether Data.py exists or not.
+If not, it sets the default Data.py as a file.
+Furthermore, it checks the correctness of Data.py and provides
+options to choose whether optional parameters should be used.
+
+Routines in this module:
+
+checkDirection(variable:str,
+               direction:str = None,
+               errorhandling:bool = False) -> str
+
+checkVariables(variable: str)
+
+checkExistVariables(variable:str) -> bool
+"""
+
+__all__ = ["checkDirection", "checkVariables"]
+
+import warnings
+import importlib
+import os
+
+if (importlib.util.find_spec("Data") is None):
+    from testsuite.Template import Data
+    warnings.warn("The file Data.py could not be found. Note that now"
+                  " optional parameters must be passed. Otherwise error"
+                  " messages will be generated or default values will"
+                  " be used. See the documentation for more"
+                  " information.")
+else:
+    import Data
+
+
+def checkDirection(variable: str = "undefined",
+                   direction: str = None,
+                   errorhandling: bool = False) -> str:
+    """
+    Checks whether the value is present in Data.py. If so, this is
+    returned if the directory exists. If there is no corresponding
+    value in Data.py, direction is returned. If both values are None
+    and no error handling is carried out, an error is raised.
+
+    Warning: if the directory does not exist, the current working
+             directory is returned.
+
+    Input:
+    -------
+    variable : str, optional
+               Is the name of the directory in Data.py
+               Note: if not set, direction must be set
+
+    direction : str, optional
+               The directory in which to search or save
+               if Data.py is not used or not set
+
+    errorhandling : bool, optional
+               Parameters for error handling only!
+               Should only be used by errorLog from Log.py.
+               If True and there is an error in the directory name,
+               this parameter sets the directory to the current working
+               directory. This ensures that at least error.log is
+               always present.
+
+    Raise:
+    -------
+    ValueError:
+        If both values (direction and Data....Direction) are None
+        and errorhandling is also False
+
+    Returns:
+    -------
+    direction : str
+                The verified input.
+                Warning: It has not yet been checked whether this
+                directory also contains the data you are looking for,
+                if this is necessary. Corresponding functions of the
+                data-reading module must be used for this.
+
+    """
+
+    val_name = "Data." + variable
+
+    # Since the variable does not necessarily have to be set,
+    # this must be checked beforehand
+    if variable in dir(Data):
+        val = eval(val_name)
```

Comment:
`eval` is kind of dangerous (but ok if you can trust user input). 
I assume you want a numeric evaluation here, ow about using [numexpr](https://github.com/pydata/numexpr)?

Reply:
I will change it in one of the next pull request. I will also take a closer look at numexpr until then.